File: core/seller.go
package core

import (
	"github.com/louisevanderlith/husk/validation"
)

type Seller struct {
	Name      string
	ContactNo string
	Email     string
	Province  string
	Town      string
	Suburb    string
}

func (s Seller) Valid() error {
	return validation.Struct(s)
}

File: lib/bodies/seller.dart
class Seller {
  final String name;
  final String contactNo;
  final String email;
  final String province;
  final String town;
  final String suburb;

  Seller(this.name, this.contactNo, this.email, this.province, this.town,
      this.suburb);

  Map<String, dynamic> toJson() {
    return {
      "Name": name,
      "ContactNo": contactNo,
      "Email": email,
      "Province": province,
      "Town": town,
      "Suburb": suburb
    };
  }
}
